jwst/saturation/x_irs2.py

```diff
@@ -76,23 +80,46 @@ def _get_irs2_parameters(input_model, n=None, r=None):
     if r is not None:
         n_ref = r
 
-    param = ReadoutParam(refout=(512 + 512 // n_norm * n_ref),
-                         n=n_norm, r=n_ref)
+    param = ReadoutParam(refout=(512 + 512 // n_norm * n_ref), n=n_norm, r=n_ref)
 
     return param
 
 
 def normal_shape(input_model, n=None, r=None, detector=None):
-    """Determine the shape of the 'normal' pixel data."""
+    """
+    Determine the shape of the 'normal' pixel data.
+
+    Parameters
+    ----------
+    input_model : DataModel
+        This is used to get the shape of the input data.
 
+    n : int or None
+        Default is None.
+        If not None, overrides the default.
+
+    r : int or None
+        Default is None.
+        If not None overrides the default.
```

Comment:
Can these be updated to say what `n` and `r` actually represent?  And instead of saying "if not None, overrides default" it might be more descriptive to say "if None, `n` is taken from the input model" or similar